### PR TITLE
Add Confluence recipe

### LINKED ISSUE #82: Confluence
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
A service to Jira Service and Conlfuence service.

**Describe alternatives you've considered**
I would like to have a service like github or trello service

**Additional context**
Thank you for everything else, you're the best!

### PR DESCRIPTION
**Pull request by vantezzen**
This PR adds a recipe for [Confluence](https://www.atlassian.com/de/software/confluence).

Closes #82

### CODE REVIEW

File: recipes/confluence/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = Ferdi => class confluence extends Ferdi {};
```
Comment (mhatvan) [inline recipes/confluence/index.js:1]: `class` name is always title case, e.g. Confluence.

But you can also simplify to: `module.exports = Ferdi => Ferdi;`

File: recipes/confluence/webview.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+module.exports = Ferdi => {
+  const getMessages = () => {
+    const unreadMessageCountElement = document.querySelector('#notifications-anchor .badge');
+    Ferdi.setBadge(unreadMessageCountElement ? Number(unreadMessageCountElement.textContent) : 0, 0);
```
Comment (mhatvan) [inline recipes/confluence/webview.js:8]: Hard to read, I would move `unreadMessageCountElement ? Number(unreadMessageCountElement.textContent) : 0` into separate variable declaration.

File: recipes/confluence/index.js (hunk 1 above)
Comment (vantezzen) [reply recipes/confluence/index.js:1]: Interesting, might need to update the automatic creator script there. The script generates the names like this - see existing recipes like https://github.com/getferdi/recipes/pull/720/files.

Thank you for the feedback, I will update the class declaration there

**Review by mhatvan — APPROVED** [review]

File: recipes/confluence/webview.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+var _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+module.exports = Ferdi => {
+  const getMessages = () => {
+    const unreadMessageCountElement = document.querySelector('#notifications-anchor .badge');
+    const unreadMessagesCount = unreadMessageCountElement ? Number(unreadMessageCountElement.textContent) : 0;
```
Comment (vraravam) [inline recipes/confluence/webview.js:8]: Please use `Ferdi.safeParseInt` - which can remove line 8

File: recipes/confluence/webview.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    Ferdi.setBadge(unreadMessagesCount, 0);
+  };
+  Ferdi.loop(getMessages);
+
+  Ferdi.injectCSS(_path.default.join(__dirname, 'service.css'));
```
Comment (vraravam) [inline recipes/confluence/webview.js:13]: If the `service.css` is empty, why include it?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/confluence/README.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# Confluence for Ferdi
```
Comment (vraravam) [inline recipes/confluence/README.md:1]: This file also seems redundant. Please remove

File: recipes/confluence/webview.js (hunk 4 above)
Comment (vantezzen) [reply recipes/confluence/webview.js:13]: This file and the Readme is generated by the creator script. As you also seem to include them (https://github.com/getferdi/recipes/pull/720/files) I thought they should remain. Might need to update the creator script if this is not wanted, I will remove them from this PR

Comment (vraravam) [reply recipes/confluence/webview.js:13]: Thanks for catching that - I have now fixed the unnecessary `service.css` in those recipes that do not use it. Please rebase from `master` - and rerun the `pnpm run package` command against your PR branch.

File: recipes/confluence/README.md (hunk 5 above)
Comment (vraravam) [reply recipes/confluence/README.md:1]: This also has been removed from the creation script.

**Comment by vraravam** [comment]
Once the PR is completed, please *squash* and merge/rebase into the `master` branch - each PR (for this repo) needs to have only a single logical commit.

**Review by vraravam — APPROVED** [review]

**Pull request merged**